system/Test/StreamFilterTrait.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Test;
+
+use CodeIgniter\Test\Filters\CITestStreamFilter;
+
+trait StreamFilterTrait
+{
+    /**
+     * @var resource|null
+     */
+    protected $streamFilterOutResource;
+
+    /**
+     * @var resource|null
+     */
+    protected $streamFilterErrResource;
```

Comment:
Any reason not to make these private?

Reply:
Thanks for the suggestion.
I took it into account and moved the trait to a separate PR https://github.com/codeigniter4/CodeIgniter4/pull/6112